packages/indexer/src/util/utils.ts

```diff
@@ -0,0 +1,53 @@
+import { CeloContract, ContractKit, newKitFromWeb3 } from '@celo/contractkit'
+import Web3 from 'web3'
+import { WEB3_PROVIDER_URL } from '../config'
+
+interface ObjectWithStringsAndUndefined {
```

Comment:
nit: Most of the things in this file I thing aren't being used

Reply:
removed!